src/components/ToggleSwitch.tsx

```diff
@@ -5,18 +5,18 @@ interface ToggleSwitchProps {
   label: string
   subtitle?: string
   onToggle: (isToggled: boolean) => void
```

Comment:
For consistency with the new prop, let's rename this to `isChecked`.

```suggestion
  onToggle: (isChecked: boolean) => void
```

Reply:
Good point! I am tempted to rename the `checked` property to `toggled`.. would that also be okay?

Reply:
Let's do `toggledOn` then? Since a switch can be toggled either way, with just `toggled` it would not be clear if it's toggled on or off.

Reply:
Perfect :ok_hand: